features/conversation-list/hooks/use-conversation-by-topic.ts

```diff
@@ -8,10 +8,6 @@ export const useConversationByTopic = (topic: ConversationTopic) => {
 
   const { data: conversations } = useConversationListQuery({
     account: account!,
```

Comment:
Just curious - what's our motivation for removing these?

Condolidation? Not needed?